### ISSUE TITLE
A swiped item should not trigger a selection

### ISSUE THREAD

**Issue by iNPUTmice** [iNPUTmice/lttrs-android#77]
When swiping and remaining in a half swiped position the long press / selection tracker kicks in.
This is a) annoying and b) can cause crashes like this:

````
03-11 21:54:09.448 30972 30972 D lttrs   : r.l.a.u.f.AbstractQueryFragment: prepare action mode for 1 selected items
03-11 21:54:09.462 30972 30972 I lttrs   : r.l.a.u.a.ThreadOverviewItemKeyProvider: attempting to get key for position 2
03-11 21:54:09.462 30972 30972 I lttrs   : r.l.a.u.a.ThreadOverviewItemKeyProvider: thread id Te35430819ce810f4
java.lang.IllegalArgumentException
        at androidx.core.util.Preconditions.checkArgument(Preconditions.java:36)
        at androidx.recyclerview.selection.DefaultSelectionTracker.anchorRange(DefaultSelectionTracker.java:288)
        at androidx.recyclerview.selection.MotionInputHandler.selectItem(MotionInputHandler.java:60)
        at androidx.recyclerview.selection.TouchInputHandler.onLongPress(TouchInputHandler.java:164)
        at androidx.recyclerview.selection.GestureRouter.onLongPress(GestureRouter.java:97)
        at android.view.GestureDetector.dispatchLongPress(GestureDetector.java:896)
        at android.view.GestureDetector.access$300(GestureDetector.java:57)
        at android.view.GestureDetector$GestureHandler.handleMessage(GestureDetector.java:322)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7660)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

**Comment by iNPUTmice** [comment]
Not fully fixed yet
```
04-22 14:39:43.716 18603 18603 D lttrs   : r.l.a.u.f.AbstractQueryFragment: prepare action mode for 1 selected items
04-22 14:39:43.729 18603 18603 I lttrs   : r.l.a.u.a.ThreadOverviewItemKeyProvider: attempting to get key for position 5
04-22 14:39:43.729 18603 18603 D AndroidRuntime: Shutting down VM
04-22 14:39:43.730 18603 18603 E AndroidRuntime: FATAL EXCEPTION: main
04-22 14:39:43.730 18603 18603 E AndroidRuntime: Process: rs.ltt.android, PID: 18603
04-22 14:39:43.730 18603 18603 E AndroidRuntime: java.lang.IndexOutOfBoundsException: Index: 5, Size: 5
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at androidx.paging.PagedList.loadAround(PagedList.java:424)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at androidx.paging.AsyncPagedListDiffer.getItem(AsyncPagedListDiffer.java:216)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at androidx.paging.PagedListAdapter.getItem(PagedListAdapter.java:176)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at rs.ltt.android.ui.adapter.ThreadOverviewAdapter.getItem(ThreadOverviewAdapter.java:201)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at rs.ltt.android.ui.adapter.ThreadOverviewItemKeyProvider.getKey(ThreadOverviewItemKeyProvider.java:46)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at rs.ltt.android.ui.adapter.ThreadOverviewItemKeyProvider.getKey(ThreadOverviewItemKeyProvider.java:31)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at androidx.recyclerview.selection.DefaultSelectionTracker.anchorRange(DefaultSelectionTracker.java:288)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at androidx.recyclerview.selection.MotionInputHandler.selectItem(MotionInputHandler.java:60)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at androidx.recyclerview.selection.TouchInputHandler.onLongPress(TouchInputHandler.java:164)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at androidx.recyclerview.selection.GestureRouter.onLongPress(GestureRouter.java:97)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at android.view.GestureDetector.dispatchLongPress(GestureDetector.java:896)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at android.view.GestureDetector.access$300(GestureDetector.java:57)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at android.view.GestureDetector$GestureHandler.handleMessage(GestureDetector.java:322)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:223)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7660)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
04-22 14:39:43.730 18603 18603 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by iNPUTmice** [comment]
Fixed in 4a2f72e2a098d09afbe0750478067374beecc085
